docs/content/tutorial/managing-your-own-io.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "Tutorial, part seven: Managing your own I/O | Dagster Docs"
+description: Learn how to use I/O managers to save your data.
+---
+
+# Tutorial, part 7: Managing your own I/O
+
+When building the dictionary or word counts, the `most_frequent_words` asset currently uses a static list of stopwords to decide what words to ignore. You’ll notice that despite the length of the list, many unhelpful words and prepositions are still not filtered out. You’d like to ignore those words. To scale this properly, you’ll use a CSV of stopwords found on the internet, use it for `most_frequent_words`, and update it before the refresh to ensure that your list of stopwords is up-to-date.
+
+While doing so, you'll take a step back and learn how to use more of Dagster's versatility without using every feature provided to you. Throughout this tutorial, you've learned multiple Dagster concepts that explain how Dagster can manage functionality on your behalf, ex. I/O managers to read and write from storage.
+
+You'll create assets to manage reading and writing from storage on your own. This is useful for multiple purposes, such as:
+
+- Your pipeline manages I/O on its own
+- You want to run a SQL query that creates or updates a table on the database.
+- Adjacent to the above point, your data is too large to fit into memory.
+- You don't want to use I/O managers.
```

Comment:
It's not clear to me how this is different than "your pipeline manages I/O on its own". Is it something like distinguishing between these two different cases?
- You're using libraries or command-line utilities that read/write to files and tables, and thus you don't have the choice of whether to use I/O managers.
- It's possible to write your pipeline using I/O managers, but you don't want to deal with this Dagster-specific concept. Perhaps because you already have working code, and you don't want to spend time refactoring it.